autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionSelect.tsx

```diff
@@ -174,6 +174,14 @@ const SectionSelect: React.FC<SectionSelectProps> = ({ id }): JSX.Element => {
         onClose={(): void => {
           setSortState({ ...sortState, sortMenuAnchor: null });
         }}
+        anchorOrigin={{
+          vertical: 'top',
+          horizontal: 'right',
+        }}
+        transformOrigin={{
+          vertical: 'top',
+          horizontal: 'right',
+        }}
```

Comment:
It seems like you accidentally duplicated this change between this PR and #536 so remove the commit